views/new.ejs

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <%- include('partials/head'); %>
+<body>
+  <%- include('partials/header', {bookTitle: 'Add A Book'}); %>
+
+  <form action="/book" method="POST">
```

Comment:
only the title and author fields should be required since you may not necessarily know or have all fields